aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -391,6 +390,12 @@ func resourceAviatrixSpokeGateway() *schema.Resource {
 				ValidateFunc: validation.IntBetween(10, 50),
 				Description:  "BGP route polling time for BGP Spoke Gateway. Unit is in seconds. Valid values are between 10 and 50.",
 			},
+			"bgp_bfd_polling_time": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     defaultBgpBfdPollingTime,
+				Description: "BGP BFD route polling time. Unit is in seconds. Valid values are between 1 and 10.",
```

Comment:
Why does this resource lack the validate function?

Reply:
Added the change